src/unix/darwin.c

```diff
@@ -333,3 +333,90 @@ void uv_free_interface_addresses(uv_interface_address_t* addresses,
 
   uv__free(addresses);
 }
+
+int uv__enable_largesocket(uv_stream_t *stream) {
+  int rmem;
+  int wmem;
+  int ret;
+  int rs;
+  int ws;
+  int rr;
+  int rw;
+  char *buf, *buf2, *pattern, *token, *token2;
+  struct stat statistics;
+  int fd = open("/etc/sysctl.conf", O_RDONLY);
```

Comment:
Instead of reading the file looking for `net.inet.tcp.sendspace` and `net.inet.tcp.recvspace` can't you just make [`sysctl()`](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man3/sysctl.3.html#//apple_ref/doc/man/3/sysctl) calls?